chainer/training/updaters/standard_updater.py

```diff
@@ -32,8 +37,10 @@ class StandardUpdater(_updater.Updater):
             extracted by the main iterator and the ``device`` option are passed
             to this function. :func:`chainer.dataset.concat_examples` is used
             by default.
-        device: Device to which the training data is sent. Negative value
-            indicates the host memory (CPU).
+        model_device (device specifier):
```

Comment:
I'm wondering if `link_device` ("link" as in `optimizer.setup`) or `target_device` ("target" as in optimizer) don't include more familiar terminologies? "model" is in my understanding a convention among Chainer-users to name their links but maybe it's used in Chainer too.